internal/auth0/organization/data_source_test.go

```diff
@@ -0,0 +1,125 @@
+
+	resource.Test(t, resource.TestCase{
+		ProviderFactories:         provider.TestFactories(httpRecorder),
+		PreventPostDestroyRefresh: true,
+		Steps: []resource.TestStep{
+			{
+				Config: template.ParseTestName(testAccGivenAnOrganizationWithConnectionsAndMembers, testName),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("auth0_connection.my_connection", "name", fmt.Sprintf("Acceptance-Test-Connection-%s", testName)),
+					resource.TestCheckResourceAttr("auth0_organization.my_organization", "name", fmt.Sprintf("test-%s", testName)),
+					resource.TestCheckResourceAttrSet("auth0_organization_connection.my_org_conn", "connection_id"),
+					resource.TestCheckResourceAttrSet("auth0_organization_connection.my_org_conn", "organization_id"),
+					resource.TestCheckResourceAttr("auth0_organization_connection.my_org_conn", "name", fmt.Sprintf("Acceptance-Test-Connection-%s", testName)),
+					resource.TestCheckResourceAttr("auth0_organization_connection.my_org_conn", "strategy", "auth0"),
+					resource.TestCheckResourceAttr("auth0_organization_connection.my_org_conn", "strategy", "auth0"),
+				),
+			},
+			{
+				Config: template.ParseTestName(testAccDataSourceOrganizationConfigByName, testName),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet("data.auth0_organization.test", "id"),
+					resource.TestCheckResourceAttr("data.auth0_organization.test", "name", fmt.Sprintf("test-%s", testName)),
+					resource.TestCheckResourceAttr("data.auth0_organization.test", "connections.#", "1"),
+					resource.TestCheckResourceAttrSet("data.auth0_organization.test", "connections.0.connection_id"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccDataSourceOrganizationByID(t *testing.T) {
+	httpRecorder := recorder.New(t)
+	testName := strings.ToLower(t.Name())
+
+	resource.Test(t, resource.TestCase{
+		ProviderFactories:         provider.TestFactories(httpRecorder),
+		PreventPostDestroyRefresh: true,
+		Steps: []resource.TestStep{
+			{
+				Config: template.ParseTestName(testAccGivenAnOrganizationWithConnectionsAndMembers, testName),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("auth0_connection.my_connection", "name", fmt.Sprintf("Acceptance-Test-Connection-%s", testName)),
+					resource.TestCheckResourceAttr("auth0_organization.my_organization", "name", fmt.Sprintf("test-%s", testName)),
+					resource.TestCheckResourceAttrSet("auth0_organization_connection.my_org_conn", "connection_id"),
+					resource.TestCheckResourceAttrSet("auth0_organization_connection.my_org_conn", "organization_id"),
+					resource.TestCheckResourceAttr("auth0_organization_connection.my_org_conn", "name", fmt.Sprintf("Acceptance-Test-Connection-%s", testName)),
+					resource.TestCheckResourceAttr("auth0_organization_connection.my_org_conn", "strategy", "auth0"),
+					resource.TestCheckResourceAttr("auth0_organization_connection.my_org_conn", "strategy", "auth0"),
```

Comment:
This one is repeated.

Reply:
Fixed [`39c6d08` (#475)](https://github.com/auth0/terraform-provider-auth0/pull/475/commits/39c6d08e8a01dc2c325f79f9c93835366453ce70)